internal/pkg/service/buffer/sink/tablesink/storage/volume/assignment/assigner.go

```diff
@@ -7,21 +7,20 @@ import (
 
 	"github.com/cespare/xxhash/v2"
 
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/sink/tablesink/storage"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/sink/tablesink/storage/volume"
 )
 
+type Assignment struct {
+	// Config is snapshot of the used assignment configuration.
+	Config Config `json:"config"`
+	// Volumes field contains assigned volumes to the file.
+	Volumes []volume.ID `json:"volumes"`
+}
+
```

Comment:
These new `Assignment` struct group snapshot of the configuration with assigned volumes IDs.